.github/workflows/static_content_deploy.yaml

```diff
@@ -9,6 +9,8 @@ on:
 env:
   bucketName: ${{ secrets.AWS_STATIC_CONTENT_BUCKET_NAME }}
   cloudFrontDeploymentId: ${{ secrets.AWS_CLOUDFRONT_DEPLOYMENT_ID }}
```

Comment:
Ty stare promenne bych vycistil. Nejsou tam uz treba.

Reply:
Jsou tam třeba dokud neověříme, že to všechno funguje :-)